recipe/meta.yaml

```diff
@@ -72,9 +63,9 @@ test:
     - cd tests
     - python -c "import daal4py"
```

Comment:
`missing_imports_or_run_test_py`: Python packages require imports or a python test file in tests.

Could you move `test/imports` (`python -c "import daal4py"`) to `run_test.py` with these imports https://github.com/intel/scikit-learn-intelex/blob/ef80ea962b05e6172df600801d883c454db84ba1/setup.py#L437-L457?
And also move all test commands to run_test.sh?